Makefile

```diff
@@ -49,9 +49,8 @@ STATIC_CSS_FILES = \
 
 LSB_RELEASE = $(shell lsb_release -cs)
 
-# The nodejs must be versioned in order to get the correct one from the PPA.
 # libfontconfig1 is required by phantom.
-SYSDEPS = nodejs=0.10.37-1chl1~utopic1 python-virtualenv g++ inotify-tools libfontconfig1
```

Comment:
We might want to pin it to 6.2.0 here so that it errors if the PPA failed to be configured for some reason.